packages/api-gateway/src/drivers/http/adapters/users/updateUser.js

```diff
@@ -0,0 +1,92 @@
+const errorHandler = require('./errorHandler');
+
+async function updateUser(req, reply) {
+  try {
+    const {
+      email,
+      avatar,
+      firstName,
+      secondName,
+      firstSurname,
+      secondSurname,
+      birthDate,
+      nationality,
+      dniId,
+      dniFrontImg,
+      dniBackImg,
+      gender,
+      phoneNumber,
+      emergencyNumber,
+      passport,
+      address: {
+        country, city, state, address, zip,
```

Comment:
Why use object destructuring here?

Reply:
Use dot notation is ok, destructuring nested elements could add complexity to the code, but it depends on conventions of the team